### DISCUSSION TITLE
Chevereto Configuring a domain name

### DISCUSSION THREAD

**Discussion by SagitTariuse** [chevereto/chevereto#37] [Q&A]
I gave Chevereto a domain name that I could use to access, but when I visited another page, the request address changed to https+ port number. Later, I set the domain name in Settings->Profile->Website, but it still didn't work

**Comment by SagitTariuse** [comment]
I use the Docker version of Chevereto

**Reply by rodber** [reply]
> Settings->Profile->Website

That controls shown website for user profile. Not the host name used by the system.

To control the URL used by the system you need to use env.

https://v4-docs.chevereto.com/application/configuration/environment.html
